server/pbench/bin/index-pbench

```diff
@@ -602,20 +820,9 @@ class ToolData(object):
             (iterseqno, itername) = iteration.split('-', 1)
         except ValueError:
             iterseqno = itername = iteration
-        try:
-            self.start_run_ts = datetime.strptime(ptb.start_run, "%Y-%m-%dT%H:%M:%S.%f")
-        except ValueError:
-            try:
-                self.start_run_ts = datetime.strptime(ptb.start_run, "%Y-%m-%dT%H:%M:%S")
-            except ValueError:
-                self.start_run_ts = None
-        try:
-            self.end_run_ts = datetime.strptime(ptb.end_run, "%Y-%m-%dT%H:%M:%S.%f")
-        except ValueError:
-            try:
-                self.end_run_ts = datetime.strptime(ptb.end_run, "%Y-%m-%dT%H:%M:%S")
-            except ValueError:
-                self.end_run_ts = None
+
+        self.start_run_ts, self.end_run_ts = ptb.gen_run_timestamps()
```

Comment:
Can you use, `super(ToolData, self).__init__(start, end)` here as well?

Reply:
Done